landscape/client/monitor/tests/test_computertags.py

```diff
@@ -13,6 +10,7 @@ def setUp(self):
         super(ComputerTagsTest, self).setUp()
         self.plugin = ComputerTags()
         self.monitor.add(self.plugin)
+        ComputerTags._argv = ['hello.py']  # For testing sys.argv
```

Comment:
My point was more to use shadowing on the instance: 
`self.plugin._argv = ["hello.py"]`
would override/shadow just that instance, without any subsiding side-effect.

Having attributes on classes and shadowing the default on test instances is a common pattern.

Reply:
Something about leveraging the python MRO which looks-up `object -> class -> superclass`

Reply:
Right, changing it to shadow the instance did break get_data() because it was using the class attribute not the instance one. To avoid these issues I used your original idea about default arguments in the __init__